examples/pxScene2d/src/Spark.cpp

```diff
@@ -718,8 +718,11 @@ if (s && (strcmp(s,"1") == 0))
   extern bool gDirtyRectsEnabled;
   rtValue dirtyRectsSetting;
   if (RT_OK == rtSettings::instance()->value("enableDirtyRects", dirtyRectsSetting))
-    gDirtyRectsEnabled = dirtyRectsSetting.toString().compare("true") == 0;
-    
+  {
+#ifdef PX_DIRTY_RECTANGLES
+      gDirtyRectsEnabled = dirtyRectsSetting.toString().compare("true") == 0;
+#endif
```

Comment:
so i added this check in case someone accidentally chooses -enableDirtyRect=true even if PX_DIRTY_RECTANGLES is OFF... I can add if (gDirtyRectsEnabled) instead ..